components/Card/index.js

```diff
@@ -48,13 +48,20 @@ export default function Card({
   image,
   title,
   hashtags,
+  favouriteIdeas,
   onToggleFavourites,
   id,
-  isFavourite,
 }) {
+  const isFavourite =
+    favouriteIdeas &&
+    Array.isArray(favouriteIdeas) &&
```

Comment:
After reviewing `_app.js` I guess you are using `Array.isArray()` as a test, because the `defaultValue` of `useLocalStorageState` is a string!? See my comment for `_app.js`.